src/service/AnnounceService.ts

```diff
@@ -265,32 +257,26 @@ export class AnnounceService {
                         logger.error(message);
                     }
                 } else {
-                    let aggregateTransaction = AggregateTransaction.createBonded(
+                    const aggregateTransaction = AggregateTransaction.createBonded(
                         deadline,
                         transactions.map((t) => t.toAggregate(mainAccount)),
                         networkType,
                         [],
-                        defaultMaxFee,
+                        maxFee,
                     );
-                    if (!providedMaxFee) {
-                        aggregateTransaction = aggregateTransaction.setMaxFeeForAggregate(minFeeMultiplier, cosigners.length - 1);
-                    }
                     const signedAggregateTransaction = bestCosigner.signTransactionWithCosignatories(
                         aggregateTransaction,
                         cosigners.filter((a) => a !== bestCosigner),
                         generationHash,
                     );
-                    let lockFundsTransaction: Transaction = LockFundsTransaction.create(
+                    const lockFundsTransaction: Transaction = LockFundsTransaction.create(
                         deadline,
                         currency.createRelative(10),
                         UInt64.fromUint(1000),
                         signedAggregateTransaction,
                         networkType,
-                        defaultMaxFee,
+                        maxFee,
```

Comment:
The max fee for a lock fund tx will be a lot smaller than aggregate tx.  Is there an issue with calculating the max fee from the minFee multiplier?

Reply:
The min fee multiplier was more problematic than expected. It creates a max fee too low that doesn't confirm. Maybe this is a bug in the server? Pretty much everyone is using the --maxFee param. 

I've seen this problem a lot https://forum.nem.io/t/symbol-supernode-programme-community-testing/30081/16